ibis/tests/all/test_temporal.py

```diff
@@ -36,14 +36,24 @@ def test_date_extract(backend, alltypes, df, attr):
 
 
 @pytest.mark.parametrize(
-    'attr', ['year', 'month', 'day', 'hour', 'minute', 'second']
+    'attr', ['year', 'month', 'day', 'epoch', 'hour', 'minute', 'second']
 )
 @pytest.mark.xfail_unsupported
 def test_timestamp_extract(backend, alltypes, df, attr):
     expr = getattr(alltypes.timestamp_col, attr)()
-    expected = getattr(df.timestamp_col.dt, attr).astype('int32')
+
+    if attr == 'epoch':
+        expected = pd.Series(
+            (pd.DatetimeIndex(df.timestamp_col) - pd.Timestamp('1970-01-01'))
+            // pd.Timedelta('1s')
```

Comment:
Not sure if it's very useful to write a test with the same implementation as the tested function. If there is no better way to test this here, I'd remove `epoch` from this parametrization, and implement a separate test where you check that `execute_epoch` returns `10` for `1970-01-01 0:00:10` or something like that.

Reply:
it is just the same implementation for pandas. so I think we should keep it here. and maybe add an extra test just for pandas. how does it sound to you?

Reply:
Makes sense, didn't realize you'd be comparing with the results of other backends.